proto/kvrpcpb.proto

```diff
@@ -226,3 +226,14 @@ message RawDeleteResponse {
     errorpb.Error region_error = 1;
     string error = 2;
 }
+
+message RawScanRequest {
+    Context context = 1;
+    bytes start_key = 2;
+    uint32 limit = 3;
+}
+
+message RawScanResponse {
+    errorpb.Error region_error = 1;
+    repeated KvPair kvs = 2;
```

Comment:
I think NextStartKey is enough if the user finds NextStartKey is none, he/she can think the scan is over.